### DISCUSSION TITLE
v1.3.0-beta.1

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#56] [Announcements]
- New Device: Adds support for the HXi and RMi series of power supplies (HID PSU).
  - Note: The minimum fan duty is 30%. When the fan power is set to 0%, control of the fan will be returned to the PSU allowing zero-RPM operation. When the fan power is set to 1% or higher, control of the fan will be returned to Fan Control.
  - Note: The LibreHardwareMonitor "PSU (Corsair)" sensor source must be disabled in Fan Control's Sensor Settings.
- Logging: Updates logging infrastructure to match the Razer plugin.

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.3.0-beta.1'>v1.3.0-beta.1</a>.</em>

**Comment by EduardoPimentel** [comment]
Will AXi series PSU support be implemented?

**Reply by EvanMulawski** [reply]
It's something I'm looking into but may not be possible. The AXi series uses a completely different protocol and a different driver - it's not as simple as communicating with an HID device.

**Reply by EduardoPimentel** [reply]
Please be sure to try this implementation. The AXi series is the best available from Corsair. Tks!

**Reply by EvanMulawski** [reply]
Just made significant progress on this actually after looking at it a bit more - see #57.
